test/models/post_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
+  def setup
+    @post = Post.new(content: "I'm a post :)")
+  end
+
+  # validate post content
+  test "post content should be present (nonblank)" do 
+    @post.content = "    " * 10
+    assert_not @post.valid?
```

Comment:
Well... this one should be valid, shouldn't it?

Reply:
Fixed 👍